packages/components/src/internal/components/samples/actions.ts

```diff
@@ -324,6 +325,26 @@ export function getSampleSelectionStorageData(selection: List<any>): Promise<Rec
     });
 }
 
+export function getSampleStorageId(sampleRowId: number): Promise<number> {
+    return new Promise((resolve, reject) => {
+        selectRows({
+            schemaName: 'inventory',
+            queryName: 'ItemSamples',
```

Comment:
Don't we have this as a constant somewhere?  Probably not in this package.

Reply:
not in this package..